core/sqf/conf/log4cxx.monitor.mon.config

```diff
@@ -22,17 +22,16 @@
 #
 
 # Define some default values that can be overridden by system properties
-trafodion.root.logger=INFO, monAppender
 trafodion.log.dir=${TRAF_HOME}/logs
 trafodion.log.filename.suffix=${TRAFODION_LOG_FILENAME_SUFFIX}
 
-# Define the root logger to the system property "trafodion.root.logger".
-log4j.rootLogger=${trafodion.root.logger}
-
 # Logging Threshold
 log4j.threshhold=ALL
 
-#
+# Define the root logger to the system property "trafodion.root.logger".
+trafodion.root.logger=FATAL
```

Comment:
I tried and the process didn't come up as I explained in my comments.   Web searching revealed that it is ok not to configure rootLogger without appender though there is no default rootLogger appender